easybuild/easyblocks/generic/pythonpackage.py

```diff
@@ -252,6 +254,13 @@ def extra_options(extra_vars=None):
             'use_setup_py_develop': [False, "Install using '%s' (deprecated)" % SETUP_PY_DEVELOP_CMD, CUSTOM],
             'zipped_egg': [False, "Install as a zipped eggs (requires use_easy_install)", CUSTOM],
         })
+        # Use PYPI_SOURCE as the default for source_urls.
+        # This even works, when the name is not all lowercase, i.e. PYPI_LOWER_SOURCE doesn't seem to be required.
```

Comment:
Erm no. "This [using PYPI_SOURCE] even works, when the name is not all lowercase, i.e. [therefore] PYPI_LOWER_SOURCE doesn't seem to be required."
So I did mean what I write here.

Example: https://pypi.python.org/packages/source/P/PyCUDA/pycuda-2020.1.tar.gz and https://pypi.python.org/packages/source/p/pycuda/pycuda-2020.1.tar.gz both work

--> PYPI_LOWER_SOURCE does not seem to be required at all

Reply:
Yes, but since PYPI_SOURCE is NOT lowercase, the sentence is strange.
And I it only works when there is an exact match at pypi to what PYPI_SOURCE is.
If you change the name of that  PyCUDA easyconfig to PYCUDA, does it still work?

Reply:
Hmm, pypi.org do indeed seem to handle random camelcasing in the path.
This one works...
https://pypi.python.org/packages/source/n/nUmPY/numpy-1.20.1.zip
And also
https://pypi.python.org/packages/source/N/NumPY/numpy-1.20.1.zip

Reply:
So a better comment may be that pypi.org handles random camelcasing of the path (but not the file) and thus we should only need PYPI_SOURCE

Reply:
Ok, updated that in https://github.com/easybuilders/easybuild-easyblocks/pull/2364/commits/972bf5322df25e623875b4325a39fc0c2a61827b
Hope that is easier to understand.

Reply:
Yes much better